minifi-assembly/pom.xml

```diff
@@ -54,21 +65,20 @@ limitations under the License.
                                     </then>
                                     <else>
                                         <echo message="Downloading Commons Daemon Windows binaries." />
-                                        <get src="https://apache.claz.org/commons/daemon/binaries/windows/commons-daemon-1.1.0-bin-windows.zip" dest="${java.io.tmpdir}/commons-daemon-1.1.0-bin-windows.zip" skipexisting="true" />
+                                        <get src="https://apache.claz.org/commons/daemon/binaries/windows/commons-daemon-${commons.daemon.version}-bin-windows.zip" dest="${java.io.tmpdir}/commons-daemon-${commons.daemon.version}-bin-windows.zip" skipexisting="true" />
```

Comment:
Do you want to use https://www.apache.org/dist/commons/daemon/binaries/windows/commons-daemon-1.2.0-bin-windows.zip instead of the mirror? Would it be more reliable against old versions going missing? I made MINIFI-502 to update the URL but it could be OBE'd by this change.

Reply:
I don't believe so and I think that is against ASF best practices.  Typically projects are instructed only to keep the latest version and then use archive.apache.org for previous releases.  Typically, the dist offerings are only for review and initial seeding.  This process seems to be reinforced as per the containing page (https://www.apache.org/dist/commons/daemon/binaries/windows/) urging the usage of a mirror.